src/librustc_codegen_llvm/back/write.rs

```diff
@@ -239,9 +239,7 @@ impl<'a> Drop for DiagnosticHandlers<'a> {
     }
 }
 
-unsafe extern "C" fn report_inline_asm(cgcx: &CodegenContext<LlvmCodegenBackend>,
-                                       msg: &str,
-                                       cookie: c_uint) {
+unsafe fn report_inline_asm(cgcx: &CodegenContext<LlvmCodegenBackend>, msg: &str, cookie: c_uint) {
```

Comment:
Is this only called once? Looks like maybe it should be inlined.

Reply:
It is called in a few places, so I haven't inlined it.